### PR TITLE
Cf camera fix again

### PR DESCRIPTION
**Pull request by thespicyboy**

### CODE REVIEW

File: Assets/Scripts/VisualizeAngle.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ public class VisualizeAngle : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
+        this.GetComponent<Transform>().localScale = this.GetComponent<Transform>().localScale * 2;
```
Comment (danpprince) [inline Assets/Scripts/VisualizeAngle.cs]: I believe Pat removed this line in favor of putting the scale in the game object in the editor, lets take it out of this branch as well

File: Assets/Scenes/Hilly plane level.unity.orig [hunk 2]
```diff
@@ -0,0 +1,3861 @@
+%YAML 1.1
```
Comment (danpprince) [inline Assets/Scenes/Hilly plane level.unity.orig]: I believe this file was created by the merge tool and we don't need it. Lets remove it and the corresponding `.meta` file

Comment (thespicyboy) [reply Assets/Scenes/Hilly plane level.unity.orig]: done

**Review by danpprince — APPROVED** [review]
Thanks for the changesssss

**Pull request merged**